static_report/src/components/ComparisonReport/CRModal.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Button,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+  Text,
+  Modal,
+  ModalHeader,
+  ModalOverlay,
+  ModalContent,
+  ModalCloseButton,
+  ModalBody,
+  ModalFooter,
+  type UseDisclosureReturn,
+} from '@chakra-ui/react';
+
+import { TestStatus } from '../shared/TestStatus';
+import { extractExpectedOrActual } from '../../utils';
+
+interface Props extends UseDisclosureReturn {
+  data?: any;
```

Comment:
Shouldn't `data` be more specific to `CR`?
I checked, and indeed our inferences are a inconsistent in our old code, so we cannot use those inferences. 

But for new code, especially single-purposed components, let's be explicit about the types where `any` or `unknown` are detected